benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -64,18 +64,45 @@ def time_broadcast(self):
 
 
 class At(Benchmark):
-    def setup(self):
+    params = (np.int8,  np.int16,  np.int32,  np.int64, np.uint8, np.uint16,
+              np.uint32, np.uint64, np.float32, np.float64)
```

Comment:
Why is this needed? Are there different code paths for the different dtypes? Our benchmark suite is getting quite large...

Reply:
No particular reason, I added it mainly temporarily because our benchmarks sometimes fluctuate widely, so I thought having multiple dtype covers will show the spread a bit.

Reply:
I think this was actually very helpful in this case, as it showed the biggest regressions are for small-sized dtypes (logical, perhaps, since then the large pointer size arrays may lead to relatively more cache pressure)